Makefile

```diff
@@ -1,9 +1,47 @@
 GOCMD=go
-VERSION=v0.3.0
+BINARY=awsnews
+BUILD_FLAGS=-ldflags="-s -w"
+PROJECT=circa10a/go-aws-news
+VERSION=0.3.0
 
 # First target for travis ci
 test:
 	$(GOCMD) test -v ./...
 
 coverage:
 	$(GOCMD) test -coverprofile=c.out ./... && go tool cover -html=c.out && rm c.out
+
+build:
+	$(GOCMD) build -o $(BINARY)
+
+compile:
+	GOOS=linux GOARCH=amd64 go build $(BUILD_FLAGS) -o bin/linux/amd64/$(BINARY)
+	GOOS=linux GOARCH=arm go build $(BUILD_FLAGS) -o bin/linux/arm/$(BINARY)
+	GOOS=linux GOARCH=arm64 go build $(BUILD_FLAGS) -o bin/linux/arm64/$(BINARY)
+	GOOS=darwin GOARCH=amd64 go build $(BUILD_FLAGS) -o bin/darwin/amd64/$(BINARY)
+	GOOS=windows GOARCH=amd64 go build $(BUILD_FLAGS) -o bin/windows/amd64/$(BINARY).exe
+
+package:
+	GOOS=linux GOARCH=amd64 tar -czf bin/$(BINARY)_$(VERSION)_linux_amd64.tar.gz -C bin/linux/amd64 $(BINARY)
+	GOOS=linux GOARCH=arm tar -czf bin/$(BINARY)_$(VERSION)_linux_arm.tar.gz -C bin/linux/arm $(BINARY)
+	GOOS=linux GOARCH=arm64 tar -czf bin/$(BINARY)_$(VERSION)_linux_arm64.tar.gz -C bin/linux/arm64 $(BINARY)
+	GOOS=darwin GOARCH=amd64 tar -czf bin/$(BINARY)_$(VERSION)_darwin_amd64.tar.gz -C bin/darwin/amd64 $(BINARY)
+	GOOS=windows GOARCH=amd64 zip -j bin/$(BINARY)_$(VERSION)_windows_amd64.zip bin/windows/amd64/$(BINARY).exe
+
+clean:
+	$(GOCMD) clean
+	rm -rf $(BINARY) bin/
+
+release: clean compile package
+
+docker-build:
+	docker build -t $(PROJECT):$(VERSION) .
+
+docker-run:
+	docker run --rm -v $(shell pwd)/config.yaml:/config.yaml $(PROJECT):$(VERSION)
+
+docker-release: docker-build
+docker-release:
```

Comment:
Don't forget to add the obligatory `latest` tag

Reply:
Actually nvm. Forgot docker hub's auto build will do that.

Reply:
If you want to do Docker Hub's auto build, then wouldn't we not have any docker stuff in Travis CI?

Reply:
Latest is pretty simple for docker hub to handle, but I don't like the way it versions the tag for you. For example on a push of tag `v1.0.0` docker hub would create `circa10a/go-aws-news:version-1.0.0`. I prefer the actual semantic version for the tag. Unless you feel differently.